### DISCUSSION TITLE
Pause/Resume projects | spend management

### DISCUSSION THREAD

**Discussion by naeem-qv** [vercel/community#6869] [Help] [answered]
### Summary

I would like to know if, when setting an upper limit of $100 using the spend management feature, all projects will automatically pause upon reaching this limit. Additionally, will these projects automatically resume when additional funds are added or at the start of a new billing cycle?

### Steps to Reproduce

.

**Comment by pawlean** [comment, marked as answer]
Hi @naeem-qv! Thanks for your patience 🙏🏼 

When you set a $100 spend limit on Vercel [[1](https://vercel.com/docs/pricing/spend-management)], all projects will automatically pause when this limit is reached. These projects will not automatically resume if you add more funds or at the start of a new billing cycle; you will need to manually resume them [[2](https://vercel.com/docs/projects/overview#pausing-a-projects)]

Let me know if you have any more questions!

[a comment by an automated account is not shown]
